### ISSUE TITLE
Deprecation warning, passing null to str_contains() - Middleware Permission

### ISSUE THREAD

**Issue by lucasjose501** [santigarcor/laratrust#651]
Not sure if this is a problem with Laratrust or Str::contains, but when I use the middleware like this in the constructor:

```
    public function __construct()
    {
        $this->middleware('permission:admin.index');
    }
```

I get this warning:
`str_contains(): Passing null to parameter #1 ($haystack) of type string is deprecated in my_app_path/vendor/laravel/framework/src/Illuminate/Support/Str.php on line 269`

To see the warning I used `LOG_DEPRECATIONS_CHANNEL=single` in the environment.

This warning happens in the function GetValuesFromParameter at LaratrustMiddleware.php when `$team` is null.
https://github.com/santigarcor/laratrust/blob/1b6fe4d695247852a82cd29a449694f985760ce9/src/Middleware/LaratrustMiddleware.php#L66

I tested locally changing the line to `Str::contains($team ?? '', ['require_all', 'guard:']) ? null : $team` and it removes the warning, but I'm not sure if this is the correct solution.

I'm using Laratrust 8.2.1 and Laravel v10.28.0

**Comment by filippotoso** [comment]
A pull request that fixes this isse has been created.
